games-action/nanosaur/nanosaur-1.4.3.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="Vintage 3D dinosaur shooter game"
+DESCRIPTION="This is Pangea Software’s Nanosaur updated to run on modern systems. Originally \
+released in 1998 as a Mac-only game, Nanosaur features a cybernetic dinosaur from the future \
+sent back in time 20 minutes before a giant asteroid hits the Earth."
+HOMEPAGE="https://pangeasoft.net/nano"
+COPYRIGHT="1998 Pangea Software, Inc."
+LICENSE="CC-BY-NC-SA-4.0"
+REVISION="1"
+
+SOURCE_URI="https://github.com/jorio/Nanosaur/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="5458ed6bf7d58facd7adf30161680546148b2a4e187c336b8af02cb864188da0"
+SOURCE_DIR="Nanosaur-$portVersion"
+PATCHES="nanosaur-$portVersion.patchset"
+ADDITIONAL_FILES="nanosaur.rdef.in"
+srcGitRev2="0581b241624d16c453e0afab57a379b443229260"
+SOURCE_URI_2="https://github.com/jorio/Pomme/archive/$srcGitRev2.tar.gz"
+CHECKSUM_SHA256_2="5e74b0061908665e7a9c710d83969aa0234b78766b05eacfa86cc3c1c0d1f666"
+SOURCE_FILENAME_2="Pomme-$srcGitRev2.tar.gz"
+SOURCE_DIR_2="Pomme-$srcGitRev2"
+PATCHES_2="nanosaur-$portVersion-source2.patchset"
+
+ARCHITECTURES="all ?x86_gcc2"
```

Comment:
```suggestion
ARCHITECTURES="all x86_gcc2"
```

Sorry, forgot to include one last change in that last set.  I was able to build and run the game on beta 3 so you can enabled that.  After this one I think it's good to merge.

Side note: I noticed the game doesn't run properly when installed to the user package directory.  Might need some more patches in the future.

Reply:
went ahead and took care of disabling x86_gcc2 to make x86 the only version of this recipe

Reply:
Great thank you, I wasn't aware it was possible to install to a user package dir. Maybe there is somewhere more appropriate for the data than next to the executable, or a way to find the actual location of the current executable programatically rather than using find_directory?

Reply:
It's certainly possibly to use find_directory() but that might mean significant changes to the code.  I haven't looked at their build system closely to see if you can pass any options, but the variables like `$dataDir` will use an absolute path that is guaranteed to exist regardless of which package directory it's installed to.(it uses the /packages/..... paths)

Reply:
After looking at the code and build system this is easiest to do with get_next_image_info() to get the app location.  I'll push a patch in a few minutes.

Reply:
Brilliant, thanks. I'll add your patch to the PR for Bugdom #6897 and future ones for Mighty Mike and Otto-Matic.